cpu/imx8-m4/cpu.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2019 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_imx8-m4
```

Comment:
```suggestion
 * @ingroup     cpu_imx8mq-m4
```
There are many IMX8 family members, not all of them have a M4, for IMX8MP it's a M7